src/main/java/com/fleotadezuta/youthprogrammanager/service/TicketTypeService.java

```diff
@@ -48,4 +49,11 @@ public Mono<TicketTypeDto> updateTicketType(String id, TicketTypeDto ticketTypeD
                 .map(ticketTypeMapper::fromTicketTypeDocumentToTicketTypeDto);
     }
 
+    public Flux<TicketTypeDto> findByName(String name) {
+        return ticketTypeRepository.findByName(name).map(ticketTypeMapper::fromTicketTypeDocumentToTicketTypeDto);
+    }
+
+    public Mono<TicketTypeDocument> findById(String id) {
```

Comment:
Erre is kellene a mapper szerintem, hogy konzisztensen `TicketTypeDto`-t adjanak vissza a service metodusok.